loader_hub/apify/dataset/README.md

```diff
@@ -1,20 +1,31 @@
 # Apify Dataset Loader
 
-Apify is a cloud platform for web scraping and data extraction,
+[Apify](https://apify.com/) is a cloud platform for web scraping and data extraction,
 which provides an [ecosystem](https://apify.com/store) of more than a thousand
 ready-made apps called *Actors* for various scraping, crawling, and extraction use cases.
 
-This loader loads documents from an [Apify dataset](https://docs.apify.com/platform/storage/dataset).
+This loader loads documents from an existing [Apify dataset](https://docs.apify.com/platform/storage/dataset).
```

Comment:
We could crosslink the `ApifyActor` loader here, to increase its discoverability

Reply:
Good point, will do